src/UI Component/Adminpanel/main.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import ViewVerified from "./Activities/verifyloan/verifyloan";
+import CurrentLoan from "./Activities/currentloan";
+import AllLoanApplications from "./Activities/allloanapplication";
+import FirstPane from "../User/firstpane";
+import AdminHome from "./adminhome";
+import LoanApprovalPage from "./Activities/loanapproval"
+import SpecialLoanRequest from "./Activities/specialloans.js";
+import PostTransactionForm from "./Activities/postloanrepayment/posttransactionform";
+
+const Button = ({ name, activeButton, onClick }) => (
+  <button
+    className={`bg-blue-400 font-bold rounded-lg text-black px-4 py-2 mr-4 ${
+      activeButton === name && "bg-blue-700"
+    }`}
+    onClick={() => onClick(name)}
+  >
+    {name}
+  </button>
+);
+
+const MainPage = () => {
+  const [activeButton, setActiveButton] = useState("adminHome");
+
+  const handleButtonClick = (buttonName) => {
+    setActiveButton(buttonName);
+  };
+
+  return (
+    <div className="mt-20">
+      <div>
+        <FirstPane />
+      </div>
+
+      <div className="flex flex-col sm:flex-row md:flex-row lg:flex-row justify-center items-center py-5 space-y-2 sm:space-x-2 sm:space-y-0">
+        <Button name="adminHome" activeButton={activeButton} onClick={handleButtonClick} />
```

Comment:
Why did you create a Button component here for reusability?

Reply:
The Button component guarantees uniform styling. By default, each button is styled with a  bg-blue-400. When a button is clicked, it displays the active button  with a  bg-blue-900. If there's a need to modify these colors in the future, it can be done that is why the button was created.